code/_onclick/other_mobs.dm

```diff
@@ -114,9 +119,15 @@
 /*
 	Animals & All Unspecified
 */
+
+// A common early return for all UnarmedAttacks
+#define UNARMED_ATTACK_COMMON \
```

Comment:
This is a crap macro, it's not obvious when reading any other code it's an early return, this is DRY gone wrong.

If you want anything to be a macro (and I'm not sure anything should be), make it the body of the if statement inside, and do the `if...return` everywhere else.

Reply:
I will reverse this and make these the start of the procs, no big deal

Reply:
How do I make this compliant with https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#no-duplicated-code, or are you asking me to ignore it this time?

Reply:
Told you how to do it over Discord, but also in this same comment:

> make it the body of the if statement inside, and do the if...return everywhere else.